server/repository/src/migrations/v1_01_01/mod.rs

```diff
@@ -29,6 +29,43 @@ impl Migration for V1_01_01 {
                 WHERE name_store_join.name_id IN (SELECT name_id FROM store WHERE store.id = name_store_join.store_id);"#
         )?;
 
+        #[cfg(not(feature = "postgres"))]
+        const INVENTORY_ADJUSTMENT_REASON_TYPE: &'static str = "TEXT";
+        #[cfg(feature = "postgres")]
+        const INVENTORY_ADJUSTMENT_REASON_TYPE: &'static str = "inventory_adjustment_type";
+        #[cfg(feature = "postgres")]
+        sql!(
+            connection,
+            r#"
+                CREATE TYPE {INVENTORY_ADJUSTMENT_REASON_TYPE} AS ENUM (
+                    'positive',
+                    'negative',
+                );
```

Comment:
These need to be UPPERCASE they come through as upper when I try to sync

```suggestion
                CREATE TYPE {INVENTORY_ADJUSTMENT_REASON_TYPE} AS ENUM (
                    'POSITIVE',
                    'NEGATIVE',
                );
```